src/plugins.star

```diff
@@ -0,0 +1,78 @@
+common = import_module("github.com/kurtosis-tech/autogpt-package/src/common.star")
+
+MAIN_BRANCH = "main"
+MASTER_BRANCH = "master"
+STDLIB_PLUGIN_REPO = "Significant-Gravitas/Auto-GPT-Plugins"
+ZIP_EXTENSION = ".zip"
+REQUIRED_ENV_VARS_KEY = "required_env_vars"
+
+plugins_map = {
+    # begin standard plugins
+    "AutoGPTTwitter": {"repository": STDLIB_PLUGIN_REPO, "branch": MASTER_BRANCH, REQUIRED_ENV_VARS_KEY: ["TW_CONSUMER_KEY", "TW_CONSUMER_SECRET", "TW_ACCESS_TOKEN", "TW_ACCESS_TOKEN_SECRET", "TW_CLIENT_ID", "TW_CLIENT_ID_SECRET"]},
+    "AutoGPTEmailPlugin": {"repository": STDLIB_PLUGIN_REPO, "branch": MASTER_BRANCH, REQUIRED_ENV_VARS_KEY: ["EMAIL_ADDRESS", "EMAIL_PASSWORD"]},
+    "AutoGPTSceneXPlugin": {"repository": STDLIB_PLUGIN_REPO, "branch": MASTER_BRANCH, REQUIRED_ENV_VARS_KEY: ["SCENEX_API_KEY"]},
+    "AutoGPTBingSearch": {"repository": STDLIB_PLUGIN_REPO, "branch": MASTER_BRANCH, REQUIRED_ENV_VARS_KEY: ["SEARCH_ENGINE", "BING_API_KEY"]}, # TODO SEARCH_ENGINE env could be auto set instead of a requirement
+    "AutoGPTNewsSearch": {"repository": STDLIB_PLUGIN_REPO, "branch": MASTER_BRANCH, REQUIRED_ENV_VARS_KEY: ["NEWSAPI_API_KEY"]},
+    "AutoGPTWikipediaSearch": {"repository": STDLIB_PLUGIN_REPO, "branch": MASTER_BRANCH, REQUIRED_ENV_VARS_KEY: []},
+    "AutoGPTApiTools": {"repository": STDLIB_PLUGIN_REPO, "branch": MASTER_BRANCH, REQUIRED_ENV_VARS_KEY: []},
+    "AutoGPTRandomValues": {"repository": STDLIB_PLUGIN_REPO, "branch": MASTER_BRANCH, REQUIRED_ENV_VARS_KEY: []},
+    "AutoGPTSpacePlugin": {"repository": STDLIB_PLUGIN_REPO, "branch": MASTER_BRANCH, REQUIRED_ENV_VARS_KEY: []},
+    "AutoGPTBaiduSearch": {"repository": STDLIB_PLUGIN_REPO, "branch": MASTER_BRANCH, REQUIRED_ENV_VARS_KEY: ["SEARCH_ENGINE", "BAIDU_COOKIE"]}, # TODO SEARCH_ENGINE env could be auto set instead of a requirement
+    "AutoGPTBluesky": {"repository": STDLIB_PLUGIN_REPO, "branch": MASTER_BRANCH, REQUIRED_ENV_VARS_KEY: ["BLUESKY_USERNAME", "BLUESKY_APP_PASSWORD"]},
+    # end of standard plugins
+    "AutoGPTAlpacaTraderPlugin": {"repository": "danikhan632/Auto-GPT-AlpacaTrader-Plugin", "branch": MASTER_BRANCH, REQUIRED_ENV_VARS_KEY:["APCA_API_KEY_ID", "APCA_API_SECRET_KEY", "APCA_IS_PAPER"]}, # TODO APCA_IS_PAPER=True can be auto set
```

Comment:
Can we not make `APCA_IS_PAPER` required for now?

https://github.com/danikhan632/Auto-GPT-AlpacaTrader-Plugin/blob/30d23005bd33755a766edf33d0d431cda30cc9e7/src/auto_gpt_alpacatrader_plugin/trader.py#L15

seems like it isn't configurable here at the moment and the author wants the default to be True

Reply:
If we can support default values in this then lets set it to True!